cmd/wallet-web/src/layouts/OIDCShareLayout.vue

```diff
@@ -0,0 +1,81 @@
+<!--
+ * Copyright SecureKey Technologies Inc. All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+-->
+
+<template>
+  <keep-alive>
+    <component
+      :is="component"
+      class="
+        overflow-hidden
+        relative
+        z-10
+        flex-grow
+        justify-start
+        items-start
+        w-full
+        h-full
+        flex flex-col
+        bg-neutrals-softWhite
+      "
+    />
+  </keep-alive>
+</template>
+
+<script>
+import { reactive, ref, watch } from 'vue';
+import WACIShare from '@/pages/WACIShare.vue';
+import { mapGetters } from 'vuex';
+import { CredentialManager } from '@trustbloc/wallet-sdk';
+import { decode } from 'js-base64';
+
+export const WACIShareLayoutStore = reactive({
+  component: WACIShare,
+});
+
+export const WACIShareLayoutMutations = {
+  setComponent(value) {
+    WACIShareLayoutStore.component = value;
+  },
+};
+
+export default {
+  name: 'OIDCShareLayout',
+  methods: {
```

Comment:
I'm curious if our linter ran on the code? Pretty sure methods should be below `setup()` and `created()`